deepchecks/checks/methodology/boosting_overfit.py

```diff
@@ -61,7 +61,7 @@ def predict_proba(self, x):
         elif self.model_class == 'CatBoostClassifier':
             return self.model.predict_proba(x, ntree_end=self.step)
         else:
-            raise DeepchecksValueError(f'Unsupported model of type: {self.model_class}')
+            raise ModelValidationError(f'Unsupported model of type: {self.model_class}')
```

Comment:
This error message (and also in lines 77 and 93) should be
"Check is relevant for Boosting models of type <list_of_supported model types>, but received model of type <model_class>"

Reply:
fixed